app/src/controllers/itemsHistory/utils.js

```diff
@@ -32,3 +32,18 @@ export const calculateMaxRowItemsCount = (history) =>
     (count, item) => (item.resources.length > count ? item.resources.length : count),
     0,
   );
+
+export const updateHistoryItemLaunchAttributes = (items = [], launch) => {
+  return items.map((item) => {
+    const resources = item.resources.map((elem) => {
```

Comment:
You can reuse already created `updateHistoryItemLaunchAttributes` from `controllers/log`.
```
const resources = updateLaunchAttributes(item.resources, launch);
```